bin/run-cloud-deployment

```diff
@@ -0,0 +1,16 @@
+#! /usr/bin/env bash
+
+# DOC: Run deploy scripts from Docker.
+
+# Usage:
+# bin/build-cloud
+# bin/run-cloud <command> <tag> <config_file>
```

Comment:
suggestion (blocking): update the comment here with the actual names of the script files